.pre-commit-config.yaml

```diff
@@ -34,6 +34,14 @@ repos:
     types: [file]
     files: (\.cmake|CMakeLists.txt)(.in)?$
 
+- repo: local
+  hooks:
+  - id: disallow-caps
+    name: Disallow improper capitalization
+    language: pygrep
+    entry: PyBind|Numpy|Cmake
```

Comment:
So this is just a list of things we don't want to see?
I guess there's no better way to implemented this? There's no way to say "this is the way these words should be capitalized, and if you do encounter a different capitalization, it's wrong" ?

Reply:
This is just a simple Python regex (it's just a custom hook in pre-commit with `pygrep` as the language); if there's a program somewhere that is smarter, we could use that instead. If the regex matches, pre-commit shows the failing line and doesn't pass.

Reply:
Well, if we encounter something, why not, later. I'll get this in now :-)